.goreleaser.yml

```diff
@@ -70,6 +70,14 @@ brews:
     install: |-
       bin.install "ionosctl"
 
+snapcrafts:
```

Comment:
is this meant to be here?

Reply:
nope, didn't notice it still being there, will remove it